src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/TemplateCollectionFunctionalTests.cs

```diff
@@ -193,11 +193,11 @@ public static IEnumerable<object[]> GetInvalidImageReference()
 
         public static IEnumerable<object[]> GetDefaultTemplatesInfo()
         {
-            yield return new object[] { "microsofthealth/fhirconverter:default", "Hl7v2" };
-            yield return new object[] { "microsofthealth/hl7v2templates:default", "Hl7v2" };
-            yield return new object[] { "microsofthealth/ccdatemplates:default", "Ccda" };
-            yield return new object[] { "microsofthealth/jsontemplates:default", "Json" };
-            yield return new object[] { "microsofthealth/stu3tor4templates:default", "Stu3ToR4" };
+            yield return new object[] { "microsofthealth/fhirconverter:default", 915 };
```

Comment:
Why do we need to change them to the fixed number? Does the fixed number often lead to failure when the number of templates increases?

Reply:
Restore to get files count, and added excluded files in unit test.